packages/core-app-api/src/apis/implementations/FetchApi/PluginProtocolResolverFetchMiddleware.test.ts

```diff
@@ -60,12 +60,6 @@ describe('PluginProtocolResolverFetchMiddleware', () => {
       'https://real.com:8080/base',
       'https://real.com:8080/base?c=d&e=f#g',
     ],
-    [
-      'plugin://username:password@x?c=d&e=f#g',
```

Comment:
This url format (with credentials) is not accepted anymore by browsers and node, so I removed it. The test won't pass with this in since we're running native node instead of node-fetch/cross-fetch.